cmd/server/main.go

```diff
@@ -82,11 +88,13 @@ func main() {
 		db = database.NewPostgresStore(*psqlDsn)
 	}
 
+	pusher := server.NewRequestPusher(db, maxItem, maxTimeOut)
```

Comment:
why can't this be `db.StartRequestPusher()`?

Reply:
can you clarify here ? Because I want to keep business and persistant layer. Don't want add batch logic inside db though